cabal-install/tests/UnitTests/Distribution/Client/VCS.hs

```diff
@@ -48,33 +50,41 @@ import UnitTests.TempTestDir (withTestDir, removeDirectoryRecursiveHack)
 --
 tests :: MTimeChange -> [TestTree]
 tests mtimeChange = map (localOption $ QuickCheckTests 10)
-  [ testGroup "git"
+  [ knownBrokenInWindows "See issue #8048" $
```

Comment:
It might be worth skipping these instead, because I think they're quite slow due to lots of disk IO.

Reply:
fair